src/test/kotlin/uk/gov/dluhc/notificationsapi/config/LocalStackContainerConfiguration.kt

```diff
@@ -69,6 +71,9 @@ class LocalStackContainerConfiguration {
         }
     }
 
+    @Bean
+    fun awsRegionProvider(): AwsRegionProvider = StaticRegionProvider(DEFAULT_REGION)
```

Comment:
with the idents added to the application-integration-test.yaml file this might not be needed anymore